bin/disco_purge_snapshots.py

```diff
@@ -40,6 +41,8 @@
 from disco_aws_automation.disco_logging import configure_logging
 
 OLD_IMAGE_DAYS = 100
+DEFAULT_KEEP_LAST = 5
+NOW = datetime.now(pytz.UTC)
```

Comment:
There is probably negligible difference in time between when `NOW` is instantiated and when the method is called. So this should be fine either way. Maybe the other way is to mock up the `datetime.now()` method in the unit test instead.